src/main/java/com/wcc/platform/controller/EventController.java

```diff
@@ -32,4 +36,12 @@ public EventController(final CmsService service) {
   public ResponseEntity<EventsPage> getEventsPage() {
     return ResponseEntity.ok(cmsService.getEvents());
```

Comment:
Are you going to add the filter in another PR? do we have ticket for that already?

Reply:
Added the Filter to the event contoller